sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/aio/_data_lake_file_client_async.py

```diff
@@ -387,10 +387,11 @@ async def set_file_expiry(
         :rtype: None
         """
         try:
-            expires_on = convert_datetime_to_rfc1123(expires_on)
+            expiration_time = convert_datetime_to_rfc1123(expires_on)
         except AttributeError:
-            expires_on = str(expires_on)
-        await self._datalake_client_for_blob_operation.path.set_expiry(expiry_options, expires_on=expires_on, **kwargs)
+            expiration_time = str(expires_on)
+        await self._datalake_client_for_blob_operation.path.set_expiry(
+            expiry_options, expires_on=expiration_time, **kwargs)
```

Comment:
nit: does this line actually exceed line length? If it does, do one parameter per line.

Reply:
It does, I'll change it to one parameter per line.